README.md

```diff
@@ -30,7 +30,7 @@ This package comes with a command called `hashauthpw`. Example usage:
 
 ```bash
 $ hashauthpw
-Usage: hashauthenticator user [secret_key [len]]
+usage: hashauthpw [-h] [--length LENGTH] username [secret_key]
```

Comment:
You can also just say `hashauthpw -h` for more as this future proofs against future changes.